specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,2383 @@
+            "x-ms-enum": {
+              "name": "SummaryAspect",
+              "modelAsString": true
+            }
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/PreBuiltTaskParameters"
+        }
+      ]
+    },
+    "AnalyzeConversationSummarizationResult": {
+      "type": "object",
+      "description": "Result for the summarization task on the conversation.",
+      "properties": {
+        "results": {
+          "$ref": "#/definitions/SummaryResult"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationJobResult"
+        }
+      ],
+      "required": [
+        "results"
+      ],
+      "x-ms-discriminator-value": "ConversationalSummarizationResults"
+    },
+    "SummaryResult": {
+      "type": "object",
+      "required": [
+        "conversations"
+      ],
+      "properties": {
+        "conversations": {
+          "type": "array",
+          "items": {
+            "allOf": [
+              {
+                "$ref": "#/definitions/ConversationsSummaryResult"
+              },
+              {
+                "$ref": "#/definitions/ConversationResultBase"
+              }
+            ]
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/PreBuiltResult"
+        }
+      ]
+    },
+    "ConversationsSummaryResult": {
+      "type": "object",
+      "required": [
+        "summaries"
+      ],
+      "properties": {
+        "summaries": {
+          "type": "array",
+          "items": {
+            "allOf": [
+              {
+                "$ref": "#/definitions/SummaryResultItem"
+              }
+            ]
+          }
+        }
+      }
+    },
+    "SummaryResultItem": {
+      "type": "object",
+      "properties": {
+        "aspect": {
+          "type": "string"
+        },
+        "text": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "aspect",
+        "text"
+      ]
+    },
+    "AnalyzeConversationJobState": {
+      "description": "Contains the status of the analyze conversations job submitted along with related statistics.",
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/JobState"
+        },
+        {
+          "$ref": "#/definitions/ConversationTasksState"
+        },
+        {
+          "$ref": "#/definitions/AnalyzeConversationJobStatistics"
+        }
+      ]
+    },
+    "AnalyzeConversationJobStatistics": {
+      "description": "Contains the statistics for the job submitted.",
+      "properties": {
+        "statistics": {
+          "$ref": "#/definitions/ConversationRequestStatistics"
+        }
+      },
+      "type": "object"
+    },
+    "ConversationTasksState": {
+      "description": "Contains the state for the tasks being executed as part of the analyze conversation job submitted.",
+      "properties": {
+        "tasks": {
+          "properties": {
```

Comment:
Some of these properties are shared with the TasksState object used in the Analyze-Text API.  We could consider factoring out the common parts to reuse between APIs in common.json.

Reply:
In next version.